flasc/utilities/optimization.py

```diff
@@ -210,6 +253,26 @@ def cost_fun(x_shift):
 
 
 def match_y_curves_by_offset(yref, ytest, dy_eval=None, angle_wrapping=True):
+    """Match two curves by finding the optimal offset.
+
+    This function is used to match two curves by finding the optimal offset
+    between the two curves. The function will minimize the mean squared error
+    between the two curves for a range of offsets.
+
+    Args:
+        yref (np.array): Reference curve.
+        ytest (np.array): Test curve.
+        dy_eval (np.array): Range of offsets to evaluate.
+            Default is None.
+        angle_wrapping (bool): Use angle wrapping for evaluation.
+            Default is True.
+
+    Returns:
+        tuple: Tuple with the following elements:
```

Comment:
```
Returns:
    dw_opt: Optimal offset
    J_opt: Optimal cost function value
```

Reply:
I think this is now in line with rest of code / google style